### DISCUSSION TITLE
DECEL light

### DISCUSSION THREAD

**Discussion by MStallen** [flybywiresim/aircraft#5250] [Help] [answered]
With very good help on this discussionpage, I managed to get the autobrakesystem working again by using the standard MSFS commands.
But I can't find to determine when the decel-light is on. What combination of parameters I should use to see that the autobrakesystem is actually engaged. The LVAR: A32NX_AUTOBRAKES_ACTIVATED shows nothing and I can't find a FSUIPC offset which shows the state. 
In the virual cockpit though, the light is showing, so there must be something that has triggered that.

**Comment by tracernz** [comment, marked as answer]
Currently WIP, so whatever you do now may break soon: https://github.com/flybywiresim/a32nx/pull/4970. Once that is merged it will be `L:A32NX_AUTOBRAKES_DECEL_LIGHT`. Prior it is `(A:AUTOBRAKES ACTIVE, Bool) or (A:REJECTED TAKEOFF BRAKES ACTIVE, Bool)`.
